vulnerabilities/importers/pypa.py

```diff
@@ -1,66 +0,0 @@
-#
-# Copyright (c) nexB Inc. and others. All rights reserved.
```

Comment:
In wonder if this should not be git moved to the new importer, to keep some history?

Reply:
Done, we're preserving git history now https://github.com/aboutcode-org/vulnerablecode/commits/ec59d7acb0726c33076367a1a4b085d01f862d3b/vulnerabilities/importers/pypa.py?browsing_rename_history=true&new_path=vulnerabilities/pipelines/pypa_importer.py&original_branch=1509-pypa-importer-pipeline